Rsupport/setupR.sh

```diff
@@ -85,7 +85,7 @@ if [ "x$BUILD_R" != "x" ]; then
 fi
 
 curl -sSL https://github.com/Crick-CancerGenomics/ascat/releases/download/v${VERSION_ASCAT}/ASCAT_${VERSION_ASCAT}.tar.gz > ascat_R.tar.gz
-curl -sSL https://github.com/cancerit/battenberg/archive/$VERSION_BB.tar.gz > BB_R.tar.gz
+curl -ssL https://github.com/cancerit/battenberg/archive/$VERSION_BB.tar.gz > BB_R.tar.gz
```

Comment:
Thought we were supposed to be switching to official repo.  Weren't they were going to do release with the chr fixes?

Reply:
This should happen for the next release of BB.